python/cudf/cudf/core/udf/groupby_typing.py

```diff
@@ -133,6 +138,24 @@ def caller(data, index, size):
     call_cuda_functions[funcname.lower()][type_key] = caller
 
 
+def _make_unary_attr(funcname):
+    class GroupUnaryReductionAttrTyping(AbstractTemplate):
+        key = f"GroupType.{funcname}"
+
+        def generic(self, args, kws):
+            for retty, inputty in call_cuda_functions[funcname.lower()].keys():
```

Comment:
Is this function supposed to return `None` or raise an error if no match is found?

Reply:
It should return `None`. This signals to the numba typing system that a match was not found.

Reply:
I'd like to have an explicit `return None` rather than an implicit no-return.